Makefile.am

```diff
@@ -353,10 +353,9 @@ dist-hook::
 	( cd $(srcdir); git ls-tree HEAD --name-only -r $(COMMITTED_DIST) || (echo $(COMMITTED_DIST) | tr ' ' '\n' ) ) | \
 		tar -C $(srcdir) -cf - -T - | tar -C $(distdir) -xf -
 	tar -C $(srcdir) -cf - --exclude='phantomjs*' node_modules/ | tar -C $(distdir) -xf -
-	tar -C $(srcdir) -cf - bower_components/ | tar -C $(distdir) -xf -
 	echo $(VERSION) > $(distdir)/.tarball
-	$(srcdir)/tools/build-copying $(BOWER) > $(distdir)/COPYING.bower
-	[ ! -d $(distdir)/tools/debian ] || $(srcdir)/tools/build-debian-copyright $(BOWER) > $(distdir)/tools/debian/copyright
+	$(srcdir)/tools/build-copying $(distdir)/bower_components > $(distdir)/COPYING.bower
+	[ ! -d $(distdir)/tools/debian ] || $(srcdir)/tools/build-debian-copyright $(distdir)/bower_components > $(distdir)/tools/debian/copyright
```

Comment:
Doesn't this require toupdate the definition of `BOWER` [here](https://github.com/stefwalter/cockpit/blob/1e2ae5c7968126604a81c10a9c1d116d7aee440f/Makefile.am#L386)? It's still being used in this Makefile.am [here](https://github.com/stefwalter/cockpit/blob/1e2ae5c7968126604a81c10a9c1d116d7aee440f/Makefile.am#L306), and also in src/base1/Makefile.am. Or should they all be updated to use `$(distdir)/bower_components` instead?